### PR TITLE
Committee Fall 2016

### PR DESCRIPTION
**Pull request by shivangipatwardhan2018**

### CODE REVIEW

File: content/pages/exec.md [hunk 1]
```diff
@@ -7,33 +7,39 @@ These are the people making things happen!
...
 
-Zarina Mamoukhouva (`zrmamouk`)
+Talia McCormick (`tdmccorm`)
 
 ## Big CSters Director ##
 
-Michelle Liauw (`nmaliauw`)
+Saadiya Desai (`sidesai`)
+
+Fatima Taj ('ftaj')
```
Comment (arshiamufti) [inline content/pages/exec.md:29]: The quotes around this are wrong.

File: content/pages/exec.md [hunk 2]
```diff
@@ -7,33 +7,39 @@ These are the people making things happen!
...
 
-Evy Kassirer (`eawkassi`)
+Anna Lorimer (`aemhlori`)
 
 ## Director of Finance ##
 
-Zarina Mamoukhouva (`zrmamouk`)
+Talia McCormick (`tdmccorm`)
 
 ## Big CSters Director ##
```
Comment (arshiamufti) [inline content/pages/exec.md]: I would rename this to "Directors". Someone should probably submit a second PR to fix the Winter 2016 exec post as well.

Comment (shivangipatwardhan2018) [reply content/pages/exec.md]: its only the one person the name highlighted in red is from last term I

File: content/pages/exec.md (hunk 1 above)
Comment (shivangipatwardhan2018) [reply content/pages/exec.md:29]: yes will fix

File: content/pages/exec.md (hunk 2 above)
Comment (arshiamufti) [reply content/pages/exec.md]: I was referring to the "Big CSters Director" heading.

Comment (shivangipatwardhan2018) [reply content/pages/exec.md]: cool will do

Comment (shivangipatwardhan2018) [reply content/pages/exec.md]: is there anything else so that I can fix it all at once?

File: content/pages/exec.md [hunk 3]
```diff
@@ -7,33 +7,39 @@ These are the people making things happen!
...
 
 ## Chair ##
 
-Arshia Mufti (`a2mufti`)
+Jenny Sun (`j89sun`)
 
 ## Secretary ##
 
-Evy Kassirer (`eawkassi`)
+Anna Lorimer (`aemhlori`)
```
Comment (arshiamufti) [inline content/pages/exec.md]: Shirley is sec this semester, not Anna.

File: content/pages/exec.md (hunk 2 above)
Comment (arshiamufti) [reply content/pages/exec.md]: I've added one more. The rest lgtm.

Comment (shivangipatwardhan2018) [reply content/pages/exec.md]: Thank you!

**Comment by evykassirer** [comment]
I'll deploy to the test site so you can double check it looks good there

**Comment by evykassirer** [comment]
The link to spring committee doesn't work but I'm not sure why. It might be a heroku thing - @fboxwala do you know what's going on?

**Comment by arshiamufti** [comment]
@evykassirer hmm maybe try re-deploying? :/

**Comment by arshiamufti** [comment]
@shivangipatwardhan2018 This should be on a separate feature branch :)

**Comment by shivangipatwardhan2018** [comment]
okay I will do it next time :)

File: content/F2016/F16-event-gooutside.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+you all can make it and bring your friends!
+
+There will be free s'mores and light dinner will be served :)
+
+## Event Details ##
+
++ **Who:** Everyone
++ **What:** Bonfire!
++ **Where:** Columbia Lake 2 Firepit
++ **When:** Thursday October 13 6:00
```
Comment (arshiamufti) [inline content/F2016/F16-event-gooutside.md]: It looks weird to me without commas? Idk this is mostly a personal preference. What do you think of Thursday, October 13, 6:00? In any case, you should change it to 6:00 pm.

Comment (arshiamufti) [reply content/F2016/F16-event-gooutside.md]: Oh, also we should specify an end time. 9 or 8 pm? Whatever you all decided at the meeting.

**Review by arshiamufti — COMMENTED** [review]
@evykassirer @fboxwala do you want to code review and merge this as soon as possible? It would be nice to have an event post up 1-2 days before the actual event 🎈

File: content/F2016/F16-event-gooutside.md (hunk 4 above)
Comment (annalorimer) [reply content/F2016/F16-event-gooutside.md]: We usually specify end time, but it's not the end of the world if we don't.

**Comment by evykassirer** [comment]
can this commit be moved to a different branch? i could walk you through it

File: content/F2016/F16-event-gooutside.md (hunk 4 above)
Comment (arshiamufti) [reply content/F2016/F16-event-gooutside.md]: Fair enough. I'm being nitpicky :P

File: content/F2016/F16-event-gooutside.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
++ **Facebook Event Page:** Click [here](https://www.facebook.com/events/534872003376714/).
```
Comment (evykassirer) [inline content/F2016/F16-event-gooutside.md]: the period looks a bit weird on https://wics-site-pr-267.herokuapp.com/ cause no other lines end in a period

and you can still make this line < 80 chars but putting `[here](https://www.facebook.com/events/534872003376714/)` on the next line :)

Comment (evykassirer) [reply content/F2016/F16-event-gooutside.md]: also I love that we link to fb from our website 🎉

File: content/F2016/F16-event-gooutside.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+title: WiCS and CSC Go Outside
+Date: 2016-10-13 18:00
```
Comment (evykassirer) [inline content/F2016/F16-event-gooutside.md]: this date is actually when you write the post, and not when the event is (pretty sure)

**Comment by fboxwala** [comment]
I'm 90% sure the links are broken because it's a heroku site and not for any good reason.  If @shivangipatwardhan2018 can split the commits to separate PRs and fix the quotes around Fatimas name I'll be happy to merge.

**Pull request merged**